docs/var.tfvars-doc.md

```diff
@@ -76,6 +76,11 @@ bootstrap                   = {instance_type    = "<bootstrap-compute-template>"
 master                      = {instance_type    = "<master-compute-template>", image_id    = "<image-uuid-rhcos>",  "count"   = 3, fixed_ips = ["<IPv4 address>", "<IPv4 address>", "<IPv4 address>"]}
 worker                      = {instance_type    = "<worker-compute-template>", image_id    = "<image-uuid-rhcos>",  "count"   = 2, fixed_ips = ["<IPv4 address>", "<IPv4 address>"]}
 ```
+To attach additional volumes to the master and worker nodes, use the optional `data_volume_count` and `data_volume_size` in master and worker variables. You can set the `data_volume_count` to any number depending on the number of volumes to be attached to each node. You can also change the `data_volume_size` as per your requirement (the `data_volume_size` is in unit GB).
```

Comment:
How about this?
To attach additional volumes to master or worker nodes, set the optional `data_volume_count` key to the number of volumes that is to be attached and the `data_volume_size` to the size (in GB) for each volume.